js-examples/test.sh

```diff
@@ -0,0 +1,25 @@
+#! /usr/bin/env bash
+#
+# Quick script that compiles and runs the samples, then cleans up.
+# Used for release testing.
+
+set -exuo pipefail
+
```

Comment:
I'd have to add $examples_dir to all the local file references, no? addressbook.capnp, addressbook.js, etc?
What about we ```cd `dirname $0` ``` ? It allows the script to get run from anywhere but the script still gets to assume its cwd.